server/sonar-server/src/main/java/org/sonar/server/computation/source/SymbolsLineReader.java

```diff
@@ -50,6 +61,18 @@ public SymbolsLineReader(Iterator<BatchReport.Symbol> symbols) {
 
   @Override
   public void read(DbFileSources.Line.Builder lineBuilder) {
+    if (!isSymbolsValid) {
+      return;
+    }
+    try {
+      processSymbols(lineBuilder);
+    } catch (RangeOffsetConverter.RangeOffsetConverterException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.internal.sonarsource.com/coding_rules#rule_key=squid%3AS1166)